Config.cmake/test_fmi3.cmake

```diff
@@ -89,9 +88,6 @@ target_link_libraries(fmi3_import_fatal_test ${FMILIBFORTEST})
 add_executable(fmi3_import_arrays_test ${FMIL_TEST_DIR}/FMI3/fmi3_import_arrays_test.c)
 target_link_libraries(fmi3_import_arrays_test ${FMILIBFORTEST})
 
-add_executable(fmi3_import_variable_types_test ${FMIL_TEST_DIR}/FMI3/fmi3_import_variable_types_test.c)
```

Comment:
This one consisted of floating point tests & unit/displayUnit tests. 

The floating point tests have been converted and moved to  `fmi3_import_variable_types_test`.

The unit/displayUnit tests served as a basis for `fmi3_import_unit_test`